### PR TITLE
[Docs] Update readme. maesh to traefik mesh

### LINKED ISSUE #1662: [Docs] Change Maesh to Traefik Mesh on readme
**Current State:**
Maesh

**Desired State:**
Traefik Mesh with updated image on readme
Can be sourced on [official site](https://traefik.io/traefik-mesh/)

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by dannylwe**
Signed-off-by: Daniel Lwetabe <acehanks@gmail.com>

**Description**
Update readme documentation.
Change Mesh to Traefik Mesh
Official Site: https://traefik.io/traefik-mesh/

This PR fixes #1662

**Notes for Reviewers**
Ready for merge
A copy of the icon has been placed in https://github.com/layer5io/meshery/tree/master/docs/assets/img/readme as requested.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by bartkalanski — DISMISSED** [review]
Good job!

**Review by leecalcote — APPROVED** [review]
Thank you, @dannylwe 👍

**Review by leecalcote — CHANGES_REQUESTED** [review]
@dannylwe will you make a copy of the icon and place in this repo under this folder? https://github.com/layer5io/meshery/tree/master/docs/assets/img/readme

Cross-linked resources go missing and we end up with broken images.

**Review by PluckyPrecious — APPROVED** [review]
LGTM 👍🏽

**Review by bartkalanski — CHANGES_REQUESTED** [review]
Now you added the icon to the repo, would you mind using that as ```src``` for the image ? You'll have to change one line of code, and once that's done, you'll be good to go!

File: README.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ See the [getting started](https://meshery.io/#getting-started) section to quickl
         </tr>
         <tr>
           <td><a href="https://github.com/layer5io/meshery-maesh">
-            <img src='https://raw.githubusercontent.com/layer5io/meshery/master/docs/assets/img/service-meshes/maesh.png' alt='Maesh Service Mesh' align="middle" hspace="5px" vspace="5px" height="30px" width="30px">Meshery adapter for Maesh</a>
+            <img src='https://raw.githubusercontent.com/layer5io/meshery/master/docs/assets/img/service-meshes/traefik.svg' alt='Traefik Service Mesh' align="middle" hspace="5px" vspace="5px" height="30px" width="30px">Meshery adapter for Traefik Mesh</a>
```
Comment (leecalcote) [inline README.md:106]: A repo-local reference. Good deal.

**Review by leecalcote — APPROVED** [review]

**Pull request merged**